MusicTaste/MusicTaste/View Controllers/MatchmakingViewController.m

```diff
@@ -110,4 +110,14 @@ - (void) receiveTestNotification:(NSNotification *) notification{
     if ([[notification name] isEqualToString:@"TestNotification"])
         NSLog (@"Successfully received the test notification!");
 }
+- (IBAction)didTapLogout:(id)sender {
+    
+    dispatch_async(dispatch_get_main_queue(), ^{
+        AppDelegate *appDelegate = (AppDelegate *)[UIApplication sharedApplication].delegate;
+        UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:nil];
+        LoginViewController *loginViewController = [storyboard instantiateViewControllerWithIdentifier:@"LoginViewController"];
+        appDelegate.window.rootViewController = loginViewController;
+        [self performSegueWithIdentifier:@"MLoginSegue" sender:nil];
+    });
```

Comment:
Segue to Login once logout button is pressed

Reply:
Does didTapLogout get called from a background thread? Why do you need `dispatch_async(dispatch_get_main_queue()` here?

Reply:
dispatch_async is not necessary that is a mistake on my end.